src/radio.rs

```diff
@@ -69,28 +72,52 @@ impl BleRadio {
 
         if ficr.overrideen.read().ble_1mbit().is_override_() {
             unsafe {
-                radio.override0.write(|w| w.override0().bits(ficr.ble_1mbit[0].read().bits()));
-                radio.override1.write(|w| w.override1().bits(ficr.ble_1mbit[1].read().bits()));
-                radio.override2.write(|w| w.override2().bits(ficr.ble_1mbit[2].read().bits()));
-                radio.override3.write(|w| w.override3().bits(ficr.ble_1mbit[3].read().bits()));
-                radio.override4.write(|w| w.override4().bits(ficr.ble_1mbit[4].read().bits()).enable().set_bit());
+                radio
+                    .override0
+                    .write(|w| w.override0().bits(ficr.ble_1mbit[0].read().bits()));
+                radio
+                    .override1
+                    .write(|w| w.override1().bits(ficr.ble_1mbit[1].read().bits()));
+                radio
+                    .override2
+                    .write(|w| w.override2().bits(ficr.ble_1mbit[2].read().bits()));
+                radio
+                    .override3
+                    .write(|w| w.override3().bits(ficr.ble_1mbit[3].read().bits()));
+                radio.override4.write(|w| {
+                    w.override4()
+                        .bits(ficr.ble_1mbit[4].read().bits())
+                        .enable()
+                        .set_bit()
+                });
             }
         }
 
         radio.mode.write(|w| w.mode().ble_1mbit());
         radio.txpower.write(|w| w.txpower().pos4d_bm());
 
         unsafe {
-            radio.pcnf1.write(|w| w
-                .maxlen().bits(37)   // no packet length limit
-                .balen().bits(3)     // 3-Byte Base Address + 1-Byte Address Prefix
-                .whiteen().set_bit() // Enable Data Whitening over PDU+CRC
+            radio.pcnf1.write(
+                |w| {
+                    w.maxlen()
+                        .bits(37) // no packet length limit
+                        .balen()
+                        .bits(3) // 3-Byte Base Address + 1-Byte Address Prefix
+                        .whiteen()
+                        .set_bit()
+                }, // Enable Data Whitening over PDU+CRC
             );
-            radio.crccnf.write(|w| w
-                .skipaddr().set_bit()   // skip address since only the S0, Length, S1 and Payload need CRC
-                .len().three()          // 3 Bytes = CRC24
+            radio.crccnf.write(
+                |w| {
+                    w.skipaddr()
+                        .set_bit() // skip address since only the S0, Length, S1 and Payload need CRC
+                        .len()
+                        .three()
+                }, // 3 Bytes = CRC24
```

Comment:
rustfmt messed up the comments here too, best to move them before the method call